File: src/BackEnd/LojaVirtual.Business/Entities/Favorito.cs
namespace LojaVirtual.Business.Entities
{
    public class Favorito
    {
        public Guid ClienteId { get; private set; }
        public Cliente Cliente { get; private set; }
        public Guid ProdutoId { get; private set; }
        public Produto Produto { get; private set; }

        public Favorito() { }

        public Favorito(Guid clienteId, Guid produtoId)
        {
            ClienteId = clienteId;
            ProdutoId = produtoId;
        }
    }
}

File: src/FrontEnd/Angular/src/app/produtos/models/favorito.ts
import { Produto } from './produto';

export interface Favorito {
  clienteId: string;
  produtoId: string;
  produto: Produto;
  nome: string;
  imagem: string;
  preco: number;
  categoria: string;
  vendedorId: string;
  vendedor: string
}